pyproject.toml

```diff
@@ -37,6 +37,7 @@ beautifulsoup4 = "^4.11.1"
 django-storages = "^1.12.3"
 boto3 = "^1.23.7"
 gunicorn = "^20.1.0"
+requests = "^2.30.0"
```

Comment:
I think we do not use this actually

Reply:
Hmm yeah that's true, I only used it for testing with a docker image